packages/flutter/test/widgets/async_test.dart

```diff
@@ -0,0 +1,274 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'dart:async';
+
+void main() {
+  group('Async smoke tests', () {
+    testWidgets('FutureBuilder', (WidgetTester tester) async {
+      await tester.pumpWidget(new FutureBuilder<String>(
+          future: new Future<String>.value('hello'),
+          builder: (BuildContext ctx, AsyncSnapshot<String> snapshot) =>
```

Comment:
We don't usually abbreviate `context` as `ctx`.  The style guide says to use complete words in variable names.